webapp/channels/src/components/post_view/show_more/show_more.test.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow} from 'enzyme';
+import {shallow, mount} from 'enzyme';
```

Comment:
Can we shy away from adding tests with enzyme? Is there any strong reason to not add the new tests for this file using react testing library?

Reply:
done